app.js

```diff
@@ -23,7 +23,7 @@ var fileUpload = require('express-fileupload');
 var dust = require('dustjs-linkedin');
 var dustHelpers = require('dustjs-helpers');
 var cons = require('consolidate');
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg" width="20"/></sub>&nbsp; Use of Hardcoded Credentials

Do not hardcode passwords in code. Found hardcoded password used in password_token.

<h6>
<a href="https://cwe.mitre.org/data/definitions/798.html">CWE-798</a> | <a href="https://cwe.mitre.org/data/definitions/259.html">CWE-259</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 574</a> | Line 31</h6>

<details>

<summary><b>Data flow: 3 steps</b></summary>

`Step 1` SOURCE https://github.com/trailblazing-dx/tiago_pr_playground/blob/be9727ad9b81d6dd7ad7c905afaf7c1607a062b5/app.js#L15

`Step 2` [example.js#51](https://github.com/owner/repo/blob/2187ce28013c2862d62c3f7fea128055deeb459a/example.js#L51)

`Step 3` SINK https://github.com/trailblazing-dx/tiago_pr_playground/blob/be9727ad9b81d6dd7ad7c905afaf7c1607a062b5/app.js#L15

</details>